### DISCUSSION TITLE
Friendly AI doesn't trigger city

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1550] [Question and answer] [answered]
True or cap?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Only players (whatever side they play) trigger the city: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/setPlayerTrigger.sqf#L30
Cheers
